Flutter/pdf-viewer/getting-started.md

```diff
@@ -113,6 +113,16 @@ tag to enable the network access in your application:
 <true/>
 ```
 
+N> Due to CORS security restriction in web application, some PDF obtained from URL is not loaded in the SfPdfViewer web platform. Kindly refer the following flutter forum,
```

Comment:
Due to CORS security restrictions in a web application, some PDFs obtained from URLs might not be loaded in the `SfPdfViewer` web platform. Kindly refer to the flutter <forum> reported on the same.

This issue can be resolved by configuring the CORS settings in the requested server or by disabling the security settings in **chrome.dart** as mentioned in the below steps: